utils/rpms/daos.spec

```diff
@@ -87,10 +87,14 @@ BuildRequires: libpsm_infinipath1
 BuildRequires: libpmemblk1
 %endif # (0%{?suse_version} >= 1315)
 %endif # (0%{?rhel} >= 7)
+
+%global min_pmdk 1.8-1
 %if (0%{?suse_version} >= 1500)
-Requires: libpmem1, libpmemobj1
+Requires: libpmem1 >= %{min_pmdk}, libpmemobj1 >= %{min_pmdk}, libpmemblk1 >= %{min_pmdk}
+%else
+Requires: libpmem >= %{min_pmdk}, libpmemobj >= %{min_pmdk}, libpmemblk >= %{min_pmdk}
 %endif
-Requires: fuse >= 3.4.2
+Requires: fuse3 >= 3.4.2
```

Comment:
The title only says pmdk

Reply:
Yeah, this was just an opportunistic change.